addons/stock/models/stock_location.py

```diff
@@ -444,6 +444,12 @@ class StockRoute(models.Model):
         'stock.warehouse', 'stock_route_warehouse', 'route_id', 'warehouse_id',
         'Warehouses', copy=False, domain="[('id', 'in', warehouse_domain_ids)]")
 
+    def copy(self, default=None):
+        default = dict(default or {})
+        if 'name' not in default:
+            default['name'] = _("%s (copy)") % self.name
```

Comment:
```suggestion
            default['name'] = _("%s (copy)", self.name)
```

Or even like this if you fell fancy :monocle_face: 
```suggestion
            default['name'] = _("%(location_name)s (copy)", location_name=self.name)
```
This later version is useful when there is multiple parameters (to be able to change the position of the terms if it's appropriate regarding the lang), with only one parameter, it only gives more context for the translators, indicates to them what they are currently translating

Reply:
:+1:

Reply:
Changed in all 4 places ;)